### DISCUSSION TITLE
Craf5 Support

### DISCUSSION THREAD

**Discussion by iuscare** [solspace/craft-calendar#258] [Questions & Answers] [answered]
Just wondering if Calendar will be available and compatible with the upcoming major release of Craft 5?

**Comment by kjmartens** [comment, marked as answer]
Hi @iuscare,

Yes, Calendar will be updated to support Craft 5, likely within the next few weeks. We were waiting for the Beta to be released before proceeding. 🙂

**Comment by smaylath** [comment]
@kjmartens thanks for the previous answer! Just checking in if there is an anticipated release date for the plugin's Craft 5 support, now that 5 is out of beta. Thank you!

**Reply by kjmartens** [reply]
Calendar 5 for Craft 5 (and 4) was released on March 26, 2024. 😊

**Reply by smaylath** [reply]
Thanks so much @kjmartens!
